packages/wallets/src/evm/connectors/smart-wallet/index.ts

```diff
@@ -32,13 +31,17 @@ export class SmartWalletConnector extends Connector<SmartWalletConnectionArgs> {
 
   constructor(config: SmartWalletConfig) {
     super();
+    if (!config.apiKey && config.thirdwebApiKey) {
```

Comment:
ugh this is annoying because thirdwebApiKey is mandatory here... so even if you want to get rid of it you cant.

actually let's make `thirdwebApiKey` optional as well here, it technically makes sense since you can pass your own bundlerURL if you want to